iothub/device/tests/Mqtt/MqttIotHubAdapterTest.cs

```diff
@@ -37,6 +39,32 @@ public void TestPopulateMessagePropertiesFromPacket_NormalMessage()
             Assert.AreEqual("MessageId1", message.SystemProperties["message-id"]);
         }
 
+        [TestMethod]
+        [ExpectedException(typeof(MessageTooLargeException), AllowDerivedTypes = false)]
+        public void TestPopulateMessagePropertiesFromMessage_TopicNameTooLong()
+        {
+            var message = new Message();
+
+            for (int i = 0; i < 65; i++)
```

Comment:
Talked offline about doing this more efficiently

Reply:
turns out, I can't set just one super long property (UriHelper doesn't allow it). seems that the loop should stay

Reply:
What is the max-len for UriHelper?